webdriver-spec.html

```diff
@@ -5128,12 +5128,12 @@
 
   <li><p>Let <var>i</var> be 0.
 
-  <li><p>Let <var>length</var> be the length of <var>character array</var>.
+  <li><p>Let <var>length</var> be the length of <var>text</var>.
 
   <li><p>While <var>i</var> &lt; <var>length</var>:
 
     <ol>
-      <li><p>Let <var>char</var> be the entry in <var>character array</var> at
+      <li><p>Let <var>char</var> be the entry in <var>text</var> at
         index <var>i</var>.
```

Comment:
There are known algorithms, provided by the unicode org, for breaking into extended grapheme clusters. I'm planning on basing "processing keys" on that.